### DISCUSSION TITLE
Nuxt 3 src folder directory

### DISCUSSION THREAD

**Discussion by k11q** [nuxt/nuxt#16418] [General]
Hi I am setting up my nuxt 3 app directory and wonder if I can put all my files in a src/ apps folder since I don't want it to be too messy in the front folder. Do I need to set up config or how can I achieve this?

**Comment by TheAlexLichter** [comment]
Just set `srcDir` in your nuxt config. See [this stackblitz](https://stackblitz.com/edit/https-github-comnuxtframeworkdiscussions8893?file=src%2Fapp.vue,nuxt.config.ts).

**Reply by k11q** [reply]
Thank you! wasn't able to find it in the v3 docs
